File: SOM.API/Models/Plantao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SOM.API.Models
{
    [Serializable]
    public class Plantao
    {
        private string _especialidade;
        private string _medico;
        private string _cremeb;
        private string _uf;
        private DateTime _dataHoraInicio;
        private DateTime _dataHoraFim;
        public string Especialidade { get => _especialidade; set => _especialidade = value; }
        public string Profissional { get => _medico; set => _medico = value; }
        public string Cremeb { get => _cremeb; set => _cremeb = value; }
        public string Uf { get => _uf; set => _uf = value; }
        public DateTime DataHoraInicio { get => _dataHoraInicio; set => _dataHoraInicio = value; }
        public DateTime DataHoraFim { get => _dataHoraFim; set => _dataHoraFim = value; }


    }
}

File: Carnaval/src/app/Models/plantao.ts
export class Plantao {

  private _especialidade: string;
  private _medico: string;
  private _cremeb: string;
  private _uf: string;
  private _dataHoraInicio: Date;
  private _dataHoraFim: Date;

  public get Especialidade(): string {
    return this._especialidade;
  }
  public set Especialidade(value: string) {
    this._especialidade = value;
  }
  public get Medico(): string {
    return this._medico;
  }
  public set Medico(value: string) {
    this._medico = value;
  }
  public get Cremeb(): string {
    return this._cremeb;
  }
  public set Cremeb(value: string) {
    this._cremeb = value;
  }
  public get Uf(): string {
    return this._uf;
  }
  public set Uf(value: string) {
    this._uf = value;
  }
  public get DataHoraInicio(): Date {
    return new Date(this._dataHoraInicio);
  }
  public set DataHoraInicio(value: Date) {
    this._dataHoraInicio = value;
  }
  public get DataHoraFim(): Date {
    return new Date(this._dataHoraFim);
  }
  public set DataHoraFim(value: Date) {
    this._dataHoraFim = value;
  }


}
